packages/pxweb2/src/app/util/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { ValueDisplayType } from '@pxweb2/pxweb2-ui';
+
+/**
+ * Returns the label text for a value based on the value display type.
+ *
+ * @param valueDisplayType - The value display type for the dimension.
+ * @param code - The code of the value.
+ * @param label - The label of the value.
+ * @returns The label text for the value.
+ */
+export function getLabelText(
+  valueDisplayType: ValueDisplayType,
+  code: string,
+  label: string,
+): string {
+  if (valueDisplayType === 'code') {
```

Comment:
Due to multiple return statements, there is no need to use else. Very nitpicky and mostly a matter of preference. So up to you 😊

It can be simplified to:

If(){Return} 
If(){Return} 
Return

Reply:
I agree, and I also prefer that so I'll update this.